### DISCUSSION TITLE
Venus OS

### DISCUSSION THREAD

**Discussion by rejeanc** [cyrils/renogy-bt#84] [Ideas]
Hi, have you tried to connect with Victron Venus OS?

**Comment by cyrils** [comment]
Never tried. But since Venus OS is linux based, there is always a possibility.

**Comment by rejeanc** [comment]
Ok thanks, will maybe try to add it as a driver in Venus OS. 

https://github.com/victronenergy/venus/wiki/howto-add-a-driver-to-Venus

**Comment by rejeanc** [comment]
I think will try it with the following : 

https://github.com/mr-manuel/venus-os_dbus-mqtt-solar-charger

Should work together, will let you know
